android/src/main/kotlin/es/antonborri/home_widget/HomeWidgetIntent.kt

```diff
@@ -21,7 +22,14 @@ object HomeWidgetLaunchIntent {
             flags = flags or PendingIntent.FLAG_IMMUTABLE
         }
 
-        return PendingIntent.getActivity(context, 0, intent, flags)
+        if (Build.VERSION.SDK_INT < 34) {
+            return PendingIntent.getActivity(context, 0, intent, flags)
+        }
+
+        val options = ActivityOptions.makeBasic()
+        options.pendingIntentBackgroundActivityStartMode = ActivityOptions.MODE_BACKGROUND_ACTIVITY_START_ALLOWED
+
+        return PendingIntent.getActivity(context, 0, intent, flags, options.toBundle())
```

Comment:
Could you instead always create the options and only add the necessary Flag when `=> 34` that way only only one return statement would need to be maintained and it is more aligned with how the flags are handled

Reply:
I was unable to do that. `ActivityOptions.makeBasic()` requires API level 23, but the min is 16.
`Call requires API level 23 (current min is 16): android.app.ActivityOptions#makeBasic`

Reply:
Alright then we go this route 👍